service-base/src/main/java/org/eclipse/hono/service/AbstractEndpoint.java

```diff
@@ -171,10 +172,23 @@ public void registerReadinessChecks(final HealthCheckHandler handler) {
      *  
      * @param spanContext The {@code SpanContext} (may be {@code null}).
      * @return The {@code DeliveryOptions}.
+     * @deprecated Use {@link #createEventBusMessageDeliveryOptions(long, SpanContext)} instead.
```

Comment:
no need to deprecate this anymore, since it is private API now ...

Reply:
ok, I will remove it.